src/sentry/mediators/token_exchange/refresher.py

```diff
@@ -26,10 +27,24 @@ class Refresher(Mediator):
     user = Param(User)
     using = router.db_for_write(User)
 
+    @sentry_sdk.trace
     def call(self):
+        sentry_sdk.set_context(
+            "token-exchange.refresh",
+            {
+                "user_id": self.user.id,
+                "install_id": self.install.id,
+                "client_id": self.client_id,
+                "org_id": self.install.organization_id,
+                "sentry_app_id": self.sentry_app.id,
+                "sentry_app_slug": self.sentry_app.slug,
+                "application_id": self.application.id,
+            },
+        )
+
         self._validate()
         self._delete_token()
-        return self._create_new_token()
+        self._create_new_token()
```

Comment:
I think we'll need this return as the serializer uses the token to make a response.

Reply:
yes, thank you for catching that 🙇‍♀️